app/fora.hoon

```diff
@@ -44,6 +44,7 @@
   |=  {pax/path sup/spur hed/@t txt/@t}
   ^-  (quip move _+>)
   :_  +>
+  ?:  =(txt '')  ~
```

Comment:
Hmm, that's true, and would be a shame. I'm fine with not doing a check here then until we have a better feedback solution.

(We should probably be doing these checks client-side anyway to prevent this from being an issue in the first place?)